courses/models_test.py

```diff
@@ -609,3 +609,17 @@ def _assert_faculty_members(obj):
         assert block.block_type == "member"
         assert block.value["name"] == "Test Faculty"
         assert block.value["description"].source == "<p>description</p>"
+
+
+def test_course_unexpired_run_dates():
+    """unexpired_run_dates should return expected value"""
+    course = CourseFactory.create()
+    start_date = now_in_utc()
+    end_date = start_date + timedelta(days=2)
+
+    CourseRunFactory.create(
+        pk=20, start_date=start_date, end_date=end_date, course=course
+    )
```

Comment:
This test case needs to create some expired course runs to ensure that we only get back unexpired runs. Consider using `create_batch` like [this example](https://github.com/mitodl/mitxpro/blob/master/courses/models_test.py#L74)

Reply:
Done.